uaclient/entitlements/fips.py

```diff
@@ -38,7 +39,7 @@ def application_status(self) -> "Tuple[status.ApplicationStatus, str]":
         if running_kernel.endswith("-fips"):
             return super_status, super_msg
         return (
-            status.ApplicationStatus.PENDING,
+            status.ApplicationStatus.ENABLED,
```

Comment:
This is a separate change from the reboot-required change. Ideally these would be split into separate commits (and perhaps separate PRs, though the PENDING->ENABLED stuff is pretty minimal).

Reply:
good thought, separated to #762 762